regulation-worker/internal/client/client.go

```diff
@@ -82,9 +87,6 @@ func (j *JobAPI) Get(ctx context.Context) (model.Job, error) {
 		pkgLogger.Debugf("obtained job: %v", job)
 		return job, nil
 
-	} else if resp.StatusCode == http.StatusNotFound {
```

Comment:
don't we need both old and new client behaviours? How are we supposed to rollout such a breaking change otherwise?

Reply:
Yeah sounds about right to me. Have both then delete the deprecated one in a later deployment, right?

Reply:
yup, I have added it back, with a note.